### PR TITLE
Stats revamp v2 update copy changes

### LINKED ISSUE #16805: [android] Stats: Update copy changes
https://docs.google.com/document/d/1OPcjKb3rbci63XbioGIEWARaVztyym3GrBCIFJXDNVY/edit

### PR DESCRIPTION
**Pull request by ravishanker**
This PR updates [copy changes as per the document here](https://docs.google.com/document/d/1OPcjKb3rbci63XbioGIEWARaVztyym3GrBCIFJXDNVY/edit)

Fixes #16805 

To test:
- Please compare the changes manually with the document as the strings may not reflect in the app until translated.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Verified manually

4. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1370,13 +1370,13 @@
     <string name="stats_insights_management_activity">Activity</string>
 
     <string name="stats_insights_likes_guide_card">⭐️ Your latest post %1$s has received %2$s likes.</string>
-    <string name="stats_insights_comments_guide_card">💡See your top contributors by tapping VIEW MORE.</string>
-    <string name="stats_insights_followers_guide_card">💡You can try leaving a comment as a gesture to encourage blog engagement in return to gain more followers.</string>
+    <string name="stats_insights_comments_guide_card">💡Tap \'VIEW MORE\' to see your top commenters.</string>
+    <string name="stats_insights_followers_guide_card">💡Commenting on other blogs is a great way to build attention and followers for your new site.</string>
```
Comment (irfano) [inline WordPress/src/main/res/values/strings.xml:1374]: There is no💡in the document.

File: WordPress/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1034,15 +1034,15 @@
     <string name="stats_management_new">New</string>
 
     <string name="stats_action_card_grow_audience_title">Grow your audience</string>
-    <string name="stats_action_card_grow_audience_message">💡We have easily digestible lessons taught by the best bloggers to grow your blog.</string>
+    <string name="stats_action_card_grow_audience_message">💡Level up your blogging faster with our expert-led &lt;i&gt;Intro to Blogging&lt;/i&gt; course.</string>
```
Comment (irfano) [inline WordPress/src/main/res/values/strings.xml:1037]: There is no💡in the document.

File: WordPress/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1034,15 +1034,15 @@
...
     <string name="stats_action_card_grow_audience_button_label">Check the course</string>
 
     <string name="stats_action_card_blogging_reminders_title">Set your blogging reminders</string>
-    <string name="stats_action_card_blogging_reminders_message">💡Set the reminder to post regularly to keep your readers engaged and attract new visitors.</string>
+    <string name="stats_action_card_blogging_reminders_message">💡Posting consistently is a great way to build your audience. Add a reminder to keep you on track.</string>
```
Comment (irfano) [inline WordPress/src/main/res/values/strings.xml:1041]: There is no💡in the document.

**Review by irfano — APPROVED** [review]
LGTM! 👍🏻 
You can remove💡 if you like. Then we're good to merge.

File: WordPress/src/main/res/values/strings.xml (hunk 1 above)
Comment (ravishanker) [reply WordPress/src/main/res/values/strings.xml:1374]: It is in the design.  Copy changes are only text

**Pull request merged**